package.json

```diff
@@ -21,15 +21,19 @@
   },
   "license": "MIT",
   "main": "lib/loglevel.js",
+  "exports": {
+    "./nomin": "./dist/loglevel.js",
+    "default": "./dist/loglevel.min.js"
+  },
```

Comment:
We do need to explore `exports`, especially for ESM support etc, but I don't think we need to do that here, we can leave that out of this PR.